packages/shared/src/components/checklist/NotificationChecklistStep.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { ReactElement } from 'react';
+import { ChecklistStepProps } from '../../lib/checklist';
+import { Button } from '../buttons/Button';
+import { ChecklistStep } from './ChecklistStep';
+import BellIcon from '../icons/Bell';
+
+const NotificationChecklistStep = (props: ChecklistStepProps): ReactElement => {
+  // TODO WT-1293-checklist-components add business logic
```

Comment:
Just a question how can we maintain that this happens when we merge this to intermediate?
I want to avoid us forgetting about it.

Reply:
Added tasks already to board.